projects/npm-tools/packages/npm-scripts/src/index.js

```diff
@@ -37,17 +38,15 @@ module.exports = async function () {
 			// Storybook is temporarily disabled until it supports webpack 5
 			// require('./scripts/storybook')();
 
-			/* eslint-disable-next-line no-console */
-			console.log(`
-			
-	WARNING:
-
-	Storybook has been temporarily disabled because it does not support 
-	webpack 5.
-
-	See https://bit.ly/35zFX4E for more information.
-
-`);
+			log(
```

Comment:
huh... is this still true? we're still using wp5? :D

Reply:
You know [I asked about removing federation related code here](https://liferay.slack.com/archives/C3JBR21HA/p1617026400057500), but was told not to. 😢

Reply:
For what it's worth, I said I'm not super into webpack right now, so I'd say we should do some cleanup and get rid of everything that we don't use. Tooling is complex enough without having to reason with things that we may or may not use.

Maybe @izaera can pick this up once you're done with TS and he's done with DatasetDisplay... 🤔

Reply:
Just ping me when your are ready @georgel-pop-lr  :point_right: https://github.com/liferay/liferay-frontend-projects/issues/480 

> You know I asked about removing federation related code here, but was told not to. :cry: 

![there there](https://media.giphy.com/media/VuWtVHkMjrz2w/giphy.gif)

Reply:
![this-is-not-over](https://user-images.githubusercontent.com/7074/113549496-d6e63480-95f1-11eb-8d47-fa81b1037037.gif)